ansible/ephemeral-devnets/playbooks/deploy-compose.yml

```diff
@@ -0,0 +1,12 @@
+---
+- name: Deploy Docker Compose
+  hosts: all
+  become: true
+  tasks:
+    - name: Copy Docker Compose file to the target server
+      copy:
+        src: "../files/{{ docker_compose_file }}"
```

Comment:
Why not use the ansible docker compose module instead ?
https://docs.ansible.com/ansible/latest/collections/community/docker/docker_compose_module.html